suite-native/device/src/components/ContinueOnTrezorScreenContent.tsx

```diff
@@ -1,38 +1,37 @@
 import { useSelector } from 'react-redux';
 
 import { selectDeviceModel } from '@suite-common/wallet-core';
-import { Box, Text } from '@suite-native/atoms';
+import { Box, BoxProps, TitleHeader } from '@suite-native/atoms';
 import { Translation, TxKeyPath } from '@suite-native/intl';
 import { DeviceModelInternal } from '@trezor/device-utils';
 import { getScreenHeight } from '@trezor/env-utils';
-import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 
 import { ConnectorImage } from './ConnectorImage';
 import { DeviceImage } from './DeviceImage';
 
 type ContinueOnTrezorScreenContentProps = {
```

Comment:
I would personally prefer composition over configuration. I mean, keeping the API of `ContinueOnTrezorScreenContent` as it was and only extracting the common part to a new component (Box with `DeviceImage` and `ConnectorImage`) and reusing that new component on `CreatePinScreen`.

Reply:
Yeh, really good point 👍 https://github.com/trezor/trezor-suite/commit/03dfab803a7ae1eb7c26280347a71667d0936999

Reply:
Also after discussion with @shenkys we keep the original images in the app and will change the image in Designs instead https://github.com/trezor/trezor-suite/pull/18329/commits/c869dffdf5c0bfd85404823c3cdeb5c41c81e92c cc: @PeKne